src/platform/forms-system/src/js/review/SSNWidget.jsx

```diff
@@ -3,11 +3,21 @@ import React from 'react';
 export default function SSNWidget({ value }) {
   if (value && value.length === 9) {
     return (
-      <span>{`${value.substr(0, 3)}-${value.substr(3, 2)}-${value.substr(
-        5,
-      )}`}</span>
+      <span
+        className="dd-privacy-hidden"
+        data-dd-action-name="Social Security number"
+      >
+        {`${value.substr(0, 3)}-${value.substr(3, 2)}-${value.substr(5)}`}
```

Comment:
[`substr` is deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr); we can replace with [`substring`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring).

Reply:
I'd rather that be part of a larger effort to update things like that. I don't want this to be a "while I'm in here"-type PR.